### ISSUE TITLE
How to compute test metric on CRC-HE-100K

### ISSUE THREAD

**Issue by dazhangyu123** [yiqings/RandStainNA#1]
I have a double about my reproduction on CRC-HE-100K. It seems that the validation set is of no use. Specifically, the validation accuracy achieves ~98% and the test accuracy achieves ~90% at the first epoch. Then, the validation accuracy will increase a little, but the test accuracy fluctuates drastically.  The test accuracy may be poor at the best validation accuracy.

**Comment by yiqings** [comment]
@dazhangyu123 Thank you for raising this helpful issue. We have observed similar phenomenon. That is the validation set may not well reflect the performance of the generalization ability (measured by test metrics). Hence, we come to a fixed epoch number to compute test acc.

**Comment by yiqings** [comment]
Through an offline discussion, we have already addressed all the concerns. Feel free to reopen the issue if you encounter other concerns.
